src/index.js

```diff
@@ -18,30 +18,25 @@ function initHealthStatement(form, signaturePad) {
 
 function createHealthStatement(form, signature, signaturePad, date = moment()) {
   const formValues = Util.serializeForm(form);
-
-  if (formValues.rememberMe === 'yes') {
-    Storage.save({
-      ...formValues,
-      signature: signaturePad.toData(),
-    });
-  } else {
-    Storage.clear();
-  }
-
   const statement = new HealthStatement(formValues);
   statement.addSignature(signature, date);
```

Comment:
You moved setting the date into the constructor, so I think it is not really needed here anymore